extensions/amp-a4a/rtc-publisher-implementation-guide.md

```diff
@@ -43,6 +43,7 @@ To use RTC, you must meet the following requirements:
 -   [PubWise](https://pubwise.io)
 -   Purch
 -   Rubicon
+-   RichAudience
```

Comment:
Nit: order

Reply:
The change has been applied in the next commit. 
Please check it.
Cheers